<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Herramienta de Optimizacion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Opciones</string>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Seleccion</string>
        </property>
        <widget class="QRadioButton" name="s_ruleta">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Ruleta</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="s_todos_un_hijo">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Todos un Hijo</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>201</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Emparejamiento</string>
        </property>
        <widget class="QRadioButton" name="e_por_orden">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Por Orden</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="e_ruleta">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Ruleta</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>201</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Cruzamiento</string>
        </property>
        <widget class="QRadioButton" name="c_cruz_simple">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Cruzamiento Simple</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="c_cruz_unif">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Cruzamiento Uniforme</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>201</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Mutacion</string>
        </property>
        <widget class="QRadioButton" name="mutacion">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>141</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Mutacion Binaria Simple</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>500</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cant. Generaciones</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="gener">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>500</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>540</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cant. Individuos</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="ind">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>540</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>411</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Problema</string>
       </property>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>361</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Datos Iniciales</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Funcion Objetivo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Restricciones</string>
         </property>
        </widget>
        <widget class="QLabel" name="fo_label">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="rest_label">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>361</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Solucion</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="optimo_label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Valor Optimo alcanzado:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Tiempo:</string>
         </property>
        </widget>
        <widget class="QLabel" name="tiempo">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>30</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>281</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Cantidad de Generaciones para el Valor Optimo alcanzado:</string>
         </property>
        </widget>
        <widget class="QLabel" name="cant_g">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>70</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Solucion:</string>
         </property>
        </widget>
        <widget class="QLabel" name="sol">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>110</y>
           <width>241</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
